pages/recording/[[...id]].tsx

```diff
@@ -66,11 +66,13 @@ function RecordingHead({ metadata }: MetadataProps) {
     description = "";
   }
 
+  const pageTitle = title ? `⏱️ ${title}` : "⏱️";
```

Comment:
How do you know that it's loading?

Reply:
This ends up being a placeholder because DevTools.tsx writes the final title once it loads:

```
  const title = recording?.title;
  const testResult = recording?.metadata?.test?.result;
  let emoji = "";
  switch (testResult) {
    case "passed":
      emoji = "✅";
      break;
    case "failed":
      emoji = "❌";
      break;
  }
  ```